### PR TITLE
add caption to burtardf slide

### PR DESCRIPTION
**Pull request by eib-fabianburtard**
Hello,

I've added my caption to the existing slides.
Can you please have a look ?

It is related to the issue #4 for the training.

Thx

### CODE REVIEW

File: _posts/0000-01-17-eib-fabianburtard.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![poptocat](https://octodex.github.com/images/poptocat.png)
 {: .center}
 
-Caption here
+This is supposed to be the best caption ever!
```
Comment (eib-fabianburtard) [inline _posts/0000-01-17-eib-fabianburtard.md:9]: Really ?

**Comment by andy2680** [comment]
could be better 🙏

**Review by gdebrito — APPROVED** [review]

**Review by jdelglise — APPROVED** [review]
Approved

**Review by florin-d — COMMENTED** [review]
@gdebrito, @anescum  Do you agree with this best ever change?

**Pull request merged**